src/modules/ROOT/pages/changelog/202209/09.json

```diff
@@ -107,10 +111,10 @@
     },
     {
       "what": [
-        "record add or update error when dedicated record revisions table is configured."
+        "user getting the error `“Could not update record: failed to complete transaction: Error 1366: Incorrect decimal value: `updated` for column `operation` at row 1”`, when adding or updating the dedicated record revision table in modules."
```

Comment:
this happened when updating records which indirectly updated the record revisions table.